containers/api/Captcha.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { useConfig } from 'react-components';
+import { getHost } from 'proton-shared/lib/helpers/url';
+import { createUrl } from 'proton-shared/lib/fetch/helpers';
+
+const Captcha = ({ token, onSubmit }) => {
+    const [style, setStyle] = useState();
+    const { API_URL } = useConfig();
+    const client = 'web';
+    const host = getHost(API_URL);
+    const url = createUrl('https://secure.protonmail.com/captcha/captcha.html', { token, client, host });
+    const src = url.toString();
+
+    const receiveMessage = (event) => {
```

Comment:
```suggestion
    const handleMessage = (event) => {
```

for consistency with our other handlers?